src/js/controller/controller.js

```diff
@@ -283,10 +285,15 @@ define([
                     viewable: viewable
                 });
                 _checkPlayOnViewable(model, viewable);
+
+                if (shouldPreload(model.get('preloaded'), viewable)) {
```

Comment:
If the only place model `model.preloaded` is read and written in here in the controller, don't use the model - add a property to the controller.